src/css/style.css

```diff
@@ -29,6 +29,10 @@
   line-height: 1.75rem;
 }
 
+.section-content-2 {
+  @apply bg-white;
```

Comment:
Do you think someone could think of tailwind's use of @apply in the same way sass uses mixins?